stripe/src/main/java/com/stripe/android/net/TelemetryClientUtil.java

```diff
@@ -0,0 +1,132 @@
+package com.stripe.android.net;
+
+import android.content.Context;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.provider.Settings;
+import android.support.annotation.NonNull;
+
+import com.stripe.android.BuildConfig;
+import com.stripe.android.util.StripeTextUtils;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.TimeZone;
+import java.util.concurrent.TimeUnit;
+
+class TelemetryClientUtil {
+
+    static Map<String, Object> createTelemetryMap(@NonNull final Context context) {
+        Map<String, Object> telemetryMap = new HashMap<>();
+        Map<String, Object> firstMap = new HashMap<>();
+        Map<String, Object> secondMap = new HashMap<>();
+        telemetryMap.put("v2", 1);
+        telemetryMap.put("tag", BuildConfig.VERSION_NAME);
+        telemetryMap.put("src", "android-sdk");
+
+        firstMap.put("c", createSingleValuePair(Locale.getDefault().toString()));
+        firstMap.put("d", createSingleValuePair(getAndroidVersionString()));
```

Comment:
on iOS, we combine device + OS version into a single platform string: `iPhone8,1 10.3.1`
should we do the same here? (e.g. concatenate `BRAND`, `MODEL` etc)
cc @jbalogh-stripe